pallets/accounts/src/lib.rs

```diff
@@ -407,18 +474,70 @@ pub mod pallet {
 			ensure!(!<EVMLinks<T>>::contains_key(&sender), Error::<T>::AccountAlreadyLinked);
 			ensure!(!<EVMLinksReverse<T>>::contains_key(eth_key), Error::<T>::AccountAlreadyLinked);
 
+			if <EthReservedTickets<T>>::contains_key(&eth_key) {
```

Comment:
Why is it possible to get tickets/nova before having a clamor account?
Why not just require one before being able to lock?

Reply:
The lock is done on Ethereum, not in our Blockchain. So nothing can actually prevent people from locking FRAG using our smart contract there, even if we create a web page to do so. 
For this reason we decided to handle this case too for now.

Reply:
Yeah but say the ETH contract requires you to specify a clamor address, wouldn't that avoid this case?

Reply:
What if the Clamor account doesn't exist though..? The caller of the smart contract could put a random address and there wouldn't be any check on Ethereum, creating a weird situation on Clamor to handle.
This is why we prefer to have people using the Link() function to link their Ethereum address to their Clamor ones..and then we do the needed check.

Reply:
Hmm yeah, was just comparing it to how ETH transfers work for example, It's up to the user to make sure they enter a correct address. (although can still be assisted later by web UI's, etc.)
But yeah the extra complexity might be worth it to avoid user error when entering the wrong address.